charts/keycloak-24/Chart.yaml

```diff
@@ -0,0 +1,36 @@
+annotations:
+  category: DeveloperTools
+  images: |
+    - name: keycloak
+      image: docker.io/bitnami/keycloak:26.3.2-debian-12-r0
+    - name: keycloak-config-cli
+      image: docker.io/bitnami/keycloak-config-cli:6.4.0-debian-12-r10
+  licenses: Apache-2.0
+  tanzuCategory: application
+apiVersion: v2
+appVersion: 26.3.2
+dependencies:
+- condition: postgresql.enabled
+  name: postgresql
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: 16.x.x
+- name: common
+  repository: oci://registry-1.docker.io/bitnamicharts
+  tags:
+  - bitnami-common
+  version: 2.x.x
+description: Keycloak is a high performance Java-based identity and access management
+  solution. It lets developers add an authentication layer to their applications with
+  minimum effort.
+home: https://bitnami.com
+icon: https://dyltqmyl993wv.cloudfront.net/assets/stacks/keycloak/img/keycloak-stack-220x234.png
+keywords:
+- keycloak
+- access-management
+maintainers:
+- name: Broadcom, Inc. All Rights Reserved.
+  url: https://github.com/bitnami/charts
```

Comment:
Should this be changed to Camunda ? + should we change the chart name?

Reply:
We can update that line, but no need to change the chart name.